Display output image in n8n

NoxTav opened the pull request:
## Description

Sometimes, APIs return raw images or REST errors in the same endpoint. Right now, if this is the case, the image is shown raw in a `json` key in n8n and needs more nodes to be shown. This PR would allow us to show the image directly in the result part.

## Effects in n8n UI

Display the image directly in the node's window as in the [HTTP Request Node](https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.httprequest/) .

## Side effects

If some use the raw image directly, it might cause them issues. I haven't investigated how it's passed to the next node.

Commit:
add image as output

devlikepro commented on src/OperationsCollector.ts:
```diff
@@ -131,24 +136,62 @@ export class BaseOperationsCollector implements OpenAPIVisitor {
+                .map(([_, data]) => (data as ResponseObject).content)
+                .map((content) => Object.keys(content!))
+                .flat()
+                .filter((contentType) => contentType.match(/image\/*/))
```
Do we need to handle only images or just regular binary data would be fine?

https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.httprequest/#n8n-binary-file

I'm not expert in how n8n handle files, but may be we could add some general logic for all binary content  types? :thinking:

devlikepro replied:
The link for send API, sorry, but the idea is the same - can all binary files be handled this way?

NoxTav replied:
All the binary files are handled the same. The key is to put the good `fileType` inside the `postReceive` function. 

Here is the list of possible keys: https://github.com/n8n-io/n8n/blob/91069f057e8e1bc32b4e2040a53e5c4546b2991f/packages/workflow/src/Interfaces.ts#L44.

I didn't make tests for HTML and text but they should work exactly the same as for the images, videos, audios and pdf.

NoxTav commented:
Hi @devlikepro, 
Just a friendly ping — I’ve addressed the feedback and updated the PR. When you have a moment, could you please take another look? Your review would be much appreciated. Thanks!

NoxTav commented:
Hi @devlikepro,
Did you have any chance to look at the issue?

devlikepro commented:
need to review and test that

JMLX42 commented:
@devlikepro any update on this? :pray:
